verilog/tools/kythe/kythe_facts.cc

```diff
@@ -68,20 +68,22 @@ bool VName::operator<(const VName& other) const {
 
 std::ostream& VName::FormatJSON(std::ostream& stream, bool debug,
                                 int indentation) const {
-  stream << "{\n";
+  // Output new line only in debug mode.
+  const std::string new_line = debug ? "\n" : "";
```

Comment:
Since it is only sometimes newline, the variable name is only sometimes right, I'd call it `separator` or `field_separator`.

(the following question is mostly showing my ignorance of how JSON is typically consumed)
Is it intentional to have an empty string instead of a 0x20 space for more even more compactness or easier shell-parsing (e.g. `awk -F,`) ?
Otherwise I'd expect that we have a 'comma space next-thing' instead of 'comma next-thing'.

Reply:
Renamed.

It's unlikely that many people consume JSON using `awk`. It's inconvenient & generally inapplicable as field separator (e.g., you could have a list). Also, there's `jq` which is breeze to use.